src/hotspot/cpu/aarch64/aarch64_sve_ad.m4

```diff
@@ -2288,3 +2287,62 @@ VECTOR_INSERT(B, iRegIorL2I, B, Register)
 VECTOR_INSERT(S, iRegIorL2I, H, Register)
 VECTOR_INSERT(I, iRegIorL2I, S, Register)
 VECTOR_INSERT(F, vRegF,      S, FloatRegister)
+
+// ------------------------------ Vector shuffle -------------------------------
+instruct loadshuffleB(vReg dst, vReg src)
+%{
+  predicate(UseSVE > 0 && n->bottom_type()->is_vect()->element_basic_type() == T_BYTE);
+  match(Set dst (VectorLoadShuffle src));
+  ins_cost(SVE_COST);
+  format %{ "sve_mov $dst, $src\n\t" %}
```

Comment:
No need to have "\n" here. And I would suggest to have more output comment just as neon implementation.

Reply:
Thank you for your review. I have fixed that.